src/lib/Watcher.js

```diff
@@ -113,17 +113,24 @@ const runPackageAndImport = widgetDirectory => {
 	_package
 		.createPackage(widgetDirectory)
 		.then(() => _import.doImport())
-		.then(() => notifySuccess())
-		.catch(error => notifyError(error));
+		.then((sequence) => {
+			let errors = sequence.filter(element => element.status === false).map(element => element.response);
+			if (errors.length > 0) {
+				return notifyError([...new Set(errors)]);
+			} else {
+				return notifySuccess();
+			}
+        })
+		.catch(error => notifyError([error]));
 }
 
 /**
  * Notify and log error message
  *
  * @return {void}
  */
-const notifyError = (error) => {
-	Notify.log(error);
+const notifyError = (errors) => {
```

Comment:
As the function now accepts multiple errors, it makes more sense to rename it to `notifyErrors`.